src/pollen.py

```diff
@@ -13,6 +13,7 @@
 from charm_state import HTTP_PORT
 from exceptions import ConfigurationWriteError, InstallError
 
+# This will be changed to 'pollen' once the upstream snap location is updated.
 SNAP_NAME = "gtrkiller-pollen"
```

Comment:
Sounds like a movie title from the Resident Evil franchise, I like it :laughing: